Upstream-Status: Inappropriate [Questone 2a support not upstream]

From 2be51d50647be1f43f0b6a0903bc7dcff47efb47 Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jonas.gorski@bisdn.de>
Date: Fri, 5 Aug 2022 11:34:17 +0000
Subject: [PATCH] cel-questone-2a: do not build optoe

No reason to use the local copy, just use the onl one that is more
recent.

Signed-off-by: Jonas Gorski <jonas.gorski@bisdn.de>
---
 .../onl/files/questone-2a/modules/builds/src/Makefile           | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/platforms/celestica/x86-64/questone-2a/modules/builds/src/Makefile b/packages/platforms/celestica/x86-64/questone-2a/modules/builds/src/Makefile
index 9bb4039..d879ecc 100644
--- a/packages/platforms/celestica/x86-64/questone-2a/modules/builds/src/Makefile
+++ b/packages/platforms/celestica/x86-64/questone-2a/modules/builds/src/Makefile
@@ -1,2 +1,2 @@
 KBUILD_CFLAGS+=-DQUESTONE2A
-obj-m := mc24lc64t.o questone2a_switchboard.o questone2a_baseboard_cpld.o sff_8436_eeprom.o optoe.o
+obj-m := mc24lc64t.o questone2a_switchboard.o questone2a_baseboard_cpld.o sff_8436_eeprom.o
-- 
2.34.1

